apps/frontend/core-dapp/src/features/position/PositionsNext.tsx

```diff
@@ -0,0 +1,213 @@
+  padding: 2px;
+  transition: opacity 200ms;
+
+  :hover {
+    opacity: 0.8;
+  }
+`
+
+type PositionProps = {
+  direction: PositionEntity['direction']
+  growthPercentage: PositionEntity['positionGrowthPercentage']
+  iconName: PositionEntity['market']['iconName']
+  name: PositionEntity['market']['name']
+  onDecrementPosition: () => void
+  onIncrementPosition: () => void
+  totalPnl: PositionEntity['totalPnl']
+  totalValue: PositionEntity['totalValue']
+  price: PositionEntity['price']
+}
+const Position: React.FC<PositionProps> = ({
+  direction,
+  growthPercentage,
+  iconName,
+  name,
+  onDecrementPosition,
+  onIncrementPosition,
+  totalPnl,
+  totalValue,
+  price,
+}) => (
+  <Flex justifyContent="start" gap={8} width="100%">
+    <Icon name={iconName} height={spacingIncrement(48)} width={spacingIncrement(48)} />
+    <Flex flexDirection="column" alignItems="start">
+      <PositionName>{name}</PositionName>
+      <Flex gap={4} alignItems="center">
+        <>
+          <PositionValue>{toUsd(totalValue)}</PositionValue>
+          {growthPercentage && (
+            <Percent
+              format={(percentValue): string => `(${percentValue})`}
+              percentagePrecision={2}
+              showPlusSign
+              styles={{ fontWeight: 'regular' }}
+              value={growthPercentage}
+            />
+          )}
+        </>
+        <Tooltip
+          overlay={
+            <Grid gridTemplateColumns="1fr 1fr" gap={4} style={{ textAlign: 'start' }}>
+              <StyledSubtitle>Current value</StyledSubtitle>
+              <p>{toUsd(totalValue)}</p>
+
+              <StyledSubtitle>Current price</StyledSubtitle>
+              <p>{toUsd(price)}</p>
+
+              <StyledSubtitle>Profit</StyledSubtitle>
+              <Flex gap={4}>
+                <>
+                  <PositionValue>{toUsd(totalPnl)}</PositionValue>
+                  {growthPercentage && (
+                    <Percent
+                      format={(percentValue): string => `(${percentValue})`}
+                      percentagePrecision={2}
+                      showPlusSign
+                      styles={{ fontWeight: 'regular' }}
+                      value={growthPercentage}
+                    />
+                  )}
+                </>
+              </Flex>
+            </Grid>
+          }
+          placement="right"
+        >
+          <Box>
+            <Icon
+              name="info"
+              height={spacingIncrement(16)}
+              width={spacingIncrement(16)}
+              color="neutral5"
+            />
+          </Box>
+        </Tooltip>
+      </Flex>
+    </Flex>
+    <Flex flexDirection="column" ml="auto" gap={6}>
+      <StyledPositionLabel positionType={direction} />
+      <Flex gap={8}>
+        <PlusButton onClick={onIncrementPosition}>
+          <Icon name="plus" color="primaryLight" width="12px" height="12px" />
+        </PlusButton>
+        <PlusButton onClick={onDecrementPosition}>
```

Comment:
Maybe rename this to "IconButton" since it's technically not a Plus here?

```suggestion
        <IconButton onClick={onDecrease}>
```